app/src/geckoRelease/java/org/mozilla/fenix/engine/GeckoProvider.kt

```diff
@@ -47,6 +47,7 @@ object GeckoProvider {
         // Use meeee.
         policy.hashCode()
 
+        @Suppress("ForbiddenComment")
```

Comment:
The TODO comment this is for is removed in the beta and nightly versions. Maybe it should just be removed here?

Reply:
Hmm.. the git history seems to have been lost for this file when we simplified our build variants. 

@liuche this API was made so that we didn't have to comment out the line and when the AC version is upgraded the underlying changes would be applied. Something seems to have changed here that I can't see just yet.

Reply:
I understand what happened now. During the variant simplification, this PR went up before merge day was complete, so this API didn't work before that.

My guess is that we committed this but forgot to uncomment this code line out.

@person808 I think this would be the right fix: remove the TODO and uncomment the `contentBlocking` line.

cc: @Amejia481

Reply:
Done. Was this supposed to be updated too?

https://github.com/mozilla-mobile/android-components/blob/c3d7281db2a6f227679b1f907f629f16a1ca7aaf/components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/ext/TrackingProtectionPolicy.kt#L24

Reply:
That's the underlying API which will be fixed during the next merge day. I believe we're using geckoview beta in Fenix releases so it should work fine after the last merge day.